Atomic/containers.py

```diff
@@ -192,7 +192,7 @@ def ps_tty(self):
                                           con_obj.id[0:max_container_id],
                                           con_obj.image_name[0:max_image_name],
                                           con_obj.name[0:max_container_name],
-                                          con_obj.command[0:max_command],
+                                          str(con_obj.command)[0:max_command],
```

Comment:
Just curious, when could the command not be a string?

Reply:
I observed that kpod commands are enclosed in [] and are interpreted differently.

Reply:
Maybe this is something we should fix in kpod images?

Reply:
yeah, ill take a look at that and then fix up here when appropriate.